src/TransformOperationExecutor.ts

```diff
@@ -440,14 +453,14 @@ export class TransformOperationExecutor {
       // apply grouping options
       if (this.options.groups && this.options.groups.length) {
         keys = keys.filter(key => {
-          const exposeMetadata = defaultMetadataStorage.findExposeMetadata(target, key);
+          const exposeMetadata = defaultMetadataStorage.findExposeMetadata(target as ClassConstructor<any>, key);
           if (!exposeMetadata || !exposeMetadata.options) return true;
 
           return this.checkGroups(exposeMetadata.options.groups);
         });
       } else {
         keys = keys.filter(key => {
-          const exposeMetadata = defaultMetadataStorage.findExposeMetadata(target, key);
+          const exposeMetadata = defaultMetadataStorage.findExposeMetadata(target as ClassConstructor<any>, key);
```

Comment:
Either we use `any` or `unknown` in the code. I am in. favor of `unknown`.

Reply:
This code is WIP, most of the castings will be removed when PR is ready to review, now I am doing it for faster iterations.

Reply:
Ok